yarn.lock

```diff
@@ -6531,6 +6532,7 @@ __metadata:
   resolution: "@storybook/components@workspace:lib/components"
   dependencies:
     "@popperjs/core": ^2.6.0
+    "@storybook/addons": 6.3.0-rc.11
```

Comment:
does this make sense? I think this is the reason the install step in the pipeline is failing

Reply:
Seems outdated now I merged in next - lets hope it fixes it, thanks for finding that man!

Reply:
Problem seems to be a hard link to @storybook/addons 6.3.0 in the yarn.lock file - only place I can find a reference to it anywhere, and that is what e2e test is complaining about